string name
int64 value
int32 priority
float32 deadline
